src/FlexCell/FlexCell.js

```diff
@@ -21,13 +21,64 @@ export const FlexCell = ({ className, children, ref, ...props }) => {
     Element = props.element;
   }
 
-  // props.width
-  if (props.width && flexCellWidth.includes(props.width)) {
-    if (typeof props.width === 'string') {
-      levers.width = `su-w-${props.width}`;
+  // props.xs to props.xxl controls width (n-of-12 columns) of the flex cell
+  // props.xs
+  if (props.xs && flexCellWidth.includes(props.xs)) {
+    if (typeof props.xs === 'string') {
+      levers.xs = `su-w-${props.xs}`;
     }
     else {
-      levers.width = `su-w-${props.width}/12`;
+      levers.xs = `su-w-${props.xs}/12`;
+    }
+  }
```

Comment:
Hmm, this could lead to a little bit of a DX issue. 

`<FlexCell xs="9" />` Wouldn't set the xs breakpoint and it would not produce an error. It would quietly not add anything. Enforcing the integer vs the string value of a number is desired but I would expect this component to throw an error if the wrong input type is passed. Also, if this value was being delivered from an API, often, it will be sent as a string instead of an integer.

Reply:
Oh interesting...that's a good point.

Reply:
Sounds like perhaps the best approach is for me to make it work for both cases - whether an integer or a string is passed?

Reply:
I'd expect one of two things to happen.
1. Throw an error if there is invalid input 
2. Output my invalid input and let the developer troubleshoot it later. 

By not outputting anything quietly, the component makes troubleshooting much more difficult. I'll leave it up to you on which direction you would like to go with it. Throwing errors or accepting loosely typed parameters is a valid solution to each. If we were writing TypeScript the strict typing of the parameter would be required and you'd likely have to throw the error.

Reply:
I was experimenting with errors, and saw that PropTypes did throw an error in the console if I tried to use a string
![Screen Shot 2021-03-08 at 5 30 02 PM](https://user-images.githubusercontent.com/42749717/110410364-9f5d8a00-803d-11eb-92c4-b9d47c3ab635.png)

Is this too quiet still? Do you think perhaps I could add a classname `su-w-ERROR` for example or is that a bad idea?